src/encoded/static/components/regulome_search.js

```diff
@@ -662,6 +668,19 @@ const filterByAllSelectedFilters = (files, facets) => {
     return newFiles;
 };
 
+const filterByOneFilter = (files, facet) => {
+    const facetFilter = facet.split('AND')[0];
+    const facetName = facet.split('AND')[1];
+    let newFiles = files;
```

Comment:
You’re not making a copy of `files` here, but it seems like it’d be easy to understand using `files.filter` below, and just using `let newFiles;` here.